### PR TITLE
DOC DataList::sort() and orderBy()

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10449

### CODE REVIEW

File: en/04_Changelogs/5.0.0.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ This is a major release and as a result there are a number of breaking API chang
 
 ### General changes {#api-general}
 
+- [DataList::sort()](api:SilverStripe\ORM\DataList) No longer accepts raw SQL. A new method `DataList::orderBy()` has been created which accepts raw SQL, though it's recommended to continue using `DataList::sort()` if possible to reduce the chance of SQL injection vulnerabilities.
```
Comment (ChloeSartorelli) [inline en/04_Changelogs/5.0.0.md]: This should be in the ORM section below.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/04_Changelogs/5.0.0.md (hunk 1 above)
Comment (emteknetnz) [reply en/04_Changelogs/5.0.0.md]: Updated

File: en/04_Changelogs/5.0.0.md [hunk 2]
```diff
@@ -164,6 +164,8 @@ This is a major release and as a result there are a number of breaking API chang
 
 ### ORM {#api-orm}
 
+- [DataList::sort()](api:SilverStripe\ORM\DataList) No longer accepts raw SQL. A new method `DataList::orderBy()` has been created which accepts raw SQL, though it's recommended to continue using `DataList::sort()` if possible to reduce the chance of SQL injection vulnerabilities.
+- Passing `null` to `DataList::sort()` i.e. `sort(null)` will now clear any existing sort values on a DataList. Passing an empty string i.e. `sort('')` or array i.e. `sort([])` will now cause an exception to be thrown.
```
Comment (ChloeSartorelli) [inline en/04_Changelogs/5.0.0.md]:
```suggestion
- Passing `null` to `DataList::sort()` i.e. `sort(null)` will now clear any existing sort values on a DataList. Passing an empty string i.e. `sort('')` or array i.e. `sort([])` will now cause an `InvalidArgumentException` to be thrown.
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/04_Changelogs/5.0.0.md (hunk 2 above)
Comment (emteknetnz) [reply en/04_Changelogs/5.0.0.md]: Updated

**Review by GuySartorelli — APPROVED** [review]
Looks good. Will merge when the framework PR is ready to merge

**Pull request merged**